### DISCUSSION TITLE
E-Mails **must** be verified when changing even if emails are'nt configured

### DISCUSSION THREAD

**Discussion by DenuxPlays** [firefly-iii/firefly-iii#8325] [Firefly III] [answered]
### Support guidelines

- [X] I've read the [support guidelines](https://github.com/firefly-iii/firefly-iii/blob/main/.github/support.md)

### I've found a bug and checked that ...

- [X] ... [the documentation](https://docs.firefly-iii.org/) does not mention anything about my problem
- [X] ... there are no open or closed issues that are related to my problem
- [X] ... it's [definitely a Firefly III issue, not me](https://github.com/firefly-iii/firefly-iii/blob/main/.github/its_you_not_me.md)

### Description

I tried changing my E-Mail address.
Then I was logged out and when I tried to log in again it says that I can't until my email was verified.
Problem is I don't and I do not want to configure a mail server.

### Debug information

Can not provide this as I cannot login into my admin user.

### Expected behaviour

Perform the email change upon verification OR even better just skip the verification if the email server isn't configured.

### Steps to reproduce

1. create an admin user
2. do not configure an email server!!!
3. change your email
4. try to log into the account

### Additional info

![image](https://github.com/firefly-iii/firefly-iii/assets/83671398/4c62a2af-6d6c-475d-825d-2a3001d9c516)

**Comment by JC5** [comment, marked as answer]
See the following documents to help you out:

- https://docs.firefly-iii.org/references/faq/firefly-iii/using/#i-changed-my-email-address-and-now-i-cant-log-in
- https://docs.firefly-iii.org/how-to/firefly-iii/advanced/notifications/#log

Set `APP_LOG_LEVEL=debug`

**Reply by DenuxPlays** [reply]
Gonna try this when I am at home.
But this shouldn't happen at all.
I think that's still an issue.

**Reply by DenuxPlays** [reply]
This will not help!
As I dont have access to my profile id and I can only get it when I am logged in.
![image](https://github.com/firefly-iii/firefly-iii/assets/83671398/13108ddc-3663-4696-9f09-096e93734d4b)

It'a a bug!
I broke my instance by using the GUI. This should not be possible!

**Reply by JC5** [reply]
Your user ID is probably 1.
